packages/publish/src/publish-command.ts

```diff
@@ -141,6 +141,13 @@ export class PublishCommand extends Command<PublishCommandOption> {
       );
     }
 
+    if (!this.options.workspaceStrictMatch) {
+      this.logger.warn(
+        'deprecation',
+        'Providing --no-workspace-strict-match is deprecated and will be removed in future version, we will make "workspace:" protocol strict matching in every case.'
```

Comment:
I'm seeing this warning even if I didn't set this flag. Is this as expected?

Reply:
@meteorlxy  oh really, no it's not expected at all, could you provide the options to help finding where that comes from?

Reply:
Uh, I was going to provide a repo and then noticed that you've fixed it. Thanks 👍

Reply:
yeah I just pushed a new release [v1.9.1](https://github.com/ghiscoding/lerna-lite/releases/tag/v1.9.1), thanks for raising the problem, it should be working as expected now, let me know if you still see it. Thanks